components/chain-switcher.tsx

```diff
@@ -0,0 +1,45 @@
+import { Card, CardContent, CardTitle } from "@/components/ui/card";
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import { useSwitchActiveWalletChain } from "thirdweb/react";
+import { baseSepolia, sepolia } from "thirdweb/chains";
+
+const ChainSwitcher = ({
+  setActiveChainChanged,
+}: {
+  setActiveChainChanged: (value: boolean) => void;
+}) => {
+  const switchChain = useSwitchActiveWalletChain();
+  return (
+    <div className="bg-background/80 w-full fixed h-full backdrop-blur-sm flex justify-center items-center">
+      <Card className="backdrop-blur-sm bg-muted-foreground/20 relative">
+        <div className="p-4">
+          <div className="flex flex-col">
+            <h1 className="text-2xl font-bold">Different Chain Detected!</h1>
+            <p className="text-sm mt-2 max-w-sm text-muted-foreground">
+              We have detected you are using a different chain. Please switch to
```

Comment:
You’re on a different network. Switch to **BaseSepolia** to keep going. 🚀
Change text to this